content/influxdb/cloud-dedicated/admin/query-system-data.md

```diff
@@ -119,6 +128,120 @@ Replace the following:
 When prompted, enter `y` to acknowledge the potential impact querying system
 tables may have on your cluster.
 
+### Optimize queries to reduce impact to your cluster
+
+Querying InfluxDB v3 system tables may impact the performance of your
+{{< product-name omit=" Clustered" >}} cluster.
+As you write data to a cluster, the number of partitions and Parquet files
+can increase to a point that impacts system table performance.
+Queries that took milliseconds with fewer files and partitions might take 10
+seconds or longer as files and partitions increase.
+
+Use the following filters to optimize your system table queries and reduce the impact on your
+cluster's performance.
+
+##### Filter by table name
+
+When querying the `system.tables`, `system.partitions`, or `system.compactor` tables, use the
+`WHERE` clause to filter by `table_name` . 
+
+```sql
+SELECT * FROM system.partitions WHERE table_name = 'TABLE_NAME'
+```
+
+##### Filter by partition key 
+
+When querying the `system.partitions` or `system.compactor` tables, use the `WHERE` clause to
+filter by `partition_key` .
+
+```sql
+SELECT * FROM system.partitions WHERE partition_key = 'PARTITION_KEY'
+```
+
+To further improve performance, use `AND` to pair `partition_key` with `table_name`--for example: 
+   
+```sql
+SELECT * 
+FROM system.partitions 
+WHERE
+  table_name = 'TABLE_NAME' 
+  AND partition_key = 'PARTITION_KEY';
+```
+
+Replace the following:
+
+- `TABLE_NAME`: the table to retrieve partitions for
+- `PARTITION_KEY`: a [partition key](/influxdb/cloud-dedicated/admin/custom-partitions/#partition-keys)
+   derived from the table's partition template.
+   The default format is `%Y-%m-%d` (for example, `2024-01-01`).
+
+##### Filter by partition ID 
+
+When querying the `system.partitions` or `system.compactor` table, use the `WHERE` clause to
+filter by `partition_id` .
+
+```sql
+SELECT * FROM system.partitions WHERE partition_id = PARTITION_ID
+```
+
+For the most optimized approach, use `AND` to pair `partition_id` with `table_name`--for example:
+
+```sql
+SELECT * 
+FROM system.partitions 
+WHERE
+  table_name = 'TABLE_NAME' 
+  AND partition_id = PARTITION_ID;
+```
+
+- `PARTITION_ID`: a [partition ID](#retrieve-a-partition-id) (int64)
+
+Although you don't need to pair `partition_id` with `table_name` (because a partition ID is unique within a cluster),
+it's the most optimized approach, _especially when you have many tables in a database_.
+
+###### Retrieve a partition ID
+
+To retrieve a partition ID, query `system.partitions` for a `table_name` and `partition_key` pair--for example:
+
+```sql
+SELECT
+  table_name,
+  partition_key,
+  partition_id 
+FROM system.partitions
+WHERE
+  table_name = 'TABLE_NAME'
+  AND partition_key = 'PARTITION_KEY';
+```
+
+The result contains the `partition_id`:
+
+| table_name | partition_key     | partition_id |
+| :--------- | :---------------- | -----------: |
+| weather    | 43 \| 2020-05-27  |         1362 |
```

Comment:
I like this example -- just select three columns to show only the necessary information. Neat! 🧹